### PR TITLE
Fix quotes in sidebar data on person_profile pages.

### PR DESCRIPTION
**Pull request by mpelzsherman**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/24177

### CODE REVIEW

File: src/site/layouts/person_profile.drupal.liquid [hunk 1]
```diff
@@ -7,7 +7,8 @@
     <main class="va-l-detail-page va-facility-page">
         <div class="usa-grid usa-grid-full">
             {% if fieldOffice.entity.entityUrl.path %}
-                {% assign sidebarData = '{"depth": 5, "link": {"label":"' | append: title | append: '","url": {"path": "/"}, "links": []}, "parent": {"label": "Leadership", "links": [], "url": {"path": "' | append: fieldOffice.entity.entityUrl.path | append: '/leadership"}}}' %}
+                {% assign formattedTitle = title | replace: '"', '&quot;' %}
+                {% assign sidebarData = '{"depth": 5, "link": {"label":"' | append: formattedTitle | append: '","url": {"path": "/"}, "links": []}, "parent": {"label": "Leadership", "links": [], "url": {"path": "' | append: fieldOffice.entity.entityUrl.path | append: '/leadership"}}}' %}
```
Comment (ncksllvn) [inline src/site/layouts/person_profile.drupal.liquid:11]: Oh my goodness, I did not know we constructed custom JSON like this from Liquid templates

**Review by ncksllvn — APPROVED** [review]

**Review by brangi — APPROVED** [review]

File: src/site/layouts/person_profile.drupal.liquid [hunk 2]
```diff
@@ -7,7 +7,8 @@
     <main class="va-l-detail-page va-facility-page">
         <div class="usa-grid usa-grid-full">
             {% if fieldOffice.entity.entityUrl.path %}
-                {% assign sidebarData = '{"depth": 5, "link": {"label":"' | append: title | append: '","url": {"path": "/"}, "links": []}, "parent": {"label": "Leadership", "links": [], "url": {"path": "' | append: fieldOffice.entity.entityUrl.path | append: '/leadership"}}}' %}
+                {% assign formattedTitle = title | replace: '"', '&quot;' %}
```
Comment (swirtSJW) [inline src/site/layouts/person_profile.drupal.liquid:10]: Is there something more generic that can be done to encapsulate all html entities?   There are a lot more that should be defended against here than just quote.

Comment (ncksllvn) [reply src/site/layouts/person_profile.drupal.liquid:10]: Yeah @kelsonic wrote an `encode` filter that might be useful here

Comment (mpelzsherman) [reply src/site/layouts/person_profile.drupal.liquid:10]: Ah, that's a good point @ncksllvn . I'll ask my team about pushing a PR to use that here instead; seems simple enough!

**Pull request merged**